src/services/goals.test.js

```diff
@@ -39,6 +54,466 @@ describe('Goals DB service', () => {
     await db.sequelize.close();
   });
 
+  describe('determineMergeGoalStatus', () => {
+    it('at least one in progress', async () => {
+      const status = determineMergeGoalStatus([
+        GOAL_STATUS.IN_PROGRESS,
+        GOAL_STATUS.CLOSED,
+      ]);
+      expect(status).toBe(GOAL_STATUS.IN_PROGRESS);
+    });
+
+    it('at least one closed', async () => {
+      const status = determineMergeGoalStatus([
+        GOAL_STATUS.CLOSED,
+        GOAL_STATUS.SUSPENDED,
+      ]);
+      expect(status).toBe(GOAL_STATUS.CLOSED);
+    });
+
+    it('at least one suspended', async () => {
+      const status = determineMergeGoalStatus([
+        GOAL_STATUS.SUSPENDED,
+        GOAL_STATUS.NOT_STARTED,
+      ]);
+      expect(status).toBe(GOAL_STATUS.SUSPENDED);
+    });
+
+    it('not started', async () => {
+      const status = determineMergeGoalStatus([
+        GOAL_STATUS.NOT_STARTED,
+        GOAL_STATUS.DRAFT,
+      ]);
+      expect(status).toBe(GOAL_STATUS.NOT_STARTED);
+    });
+
+    it('DRAFT', async () => {
+      const status = determineMergeGoalStatus([
+        GOAL_STATUS.DRAFT,
+        GOAL_STATUS.DRAFT,
+      ]);
+      expect(status).toBe(GOAL_STATUS.DRAFT);
+    });
+  });
+
+  describe('mergeGoals', () => {
+    let recipient;
+    let grantOne;
+    let grantTwo;
+    let grantThree;
+    let report;
+    let template;
+    let topic;
+    let goalOne;
+    let goalTwo;
+    let goalThree;
+
+    let objectiveOneForGoalOne;
+
+    const fileName = faker.system.fileName();
+    const resourceUrl = faker.internet.url();
+
+    beforeEach(async () => {
+      // Recipient.
+      recipient = await Recipient.create({
```

Comment:
Not sure if you want to include a recipient, grant, etc. that should NOT be updated. And just double check.

Reply:
recipients and grants aren't updated at all, but I added a goal and an objective that weren't updated